l3kernel/l3doc.dtx

```diff
@@ -238,7 +238,7 @@ and all files in that bundle must be distributed together.
 %
 % \DescribeOption{kernel}
 % Determines whether \pkg{l3doc} treats |\__kernel_| commands and
-% |\(c|g|l)__kernel_| variables as allowable in code. In general,
+% "\(c|g|l)__kernel_" variables as allowable in code. In general,
```

Comment:
before it was simply wrong, now it is just odd, because everything is in typewriter. I guess most people will understand though, but perhaps one should do the extra step and use normal text font on the parentheses or just avoid this altogether and write them separately. Dunno

Reply:
Maybe just rewrite this to "treats functions and variables under `kernel` module".

Reply:
or something like this, yes. One may might still want to mention `|__kernel_|` in that sentence as that is just highlighting what "kernel module" means as this is correct for all of them and might make it clearer what is meant.

Reply:
Oh, it should be "internal (functions and variables)".